component/azstorage/block_blob.go

```diff
@@ -666,6 +666,8 @@ func (bb *BlockBlob) getBlobAttr(blobInfo *container.BlobItem) (*internal.ObjAtt
 		Flags:  internal.NewFileBitMap(),
 		MD5:    blobInfo.Properties.ContentMD5,
 		ETag:   sanitizeEtag(blobInfo.Properties.ETag),
+		GID:    blobInfo.Metadata["gid"],
```

Comment:
We shall not use random names, check what azcopy uses and we shall be using the same key name in metadata to cross tool compatibility.